benchpress/benchpress/plugins/parsers/generic.py

```diff
@@ -8,16 +8,22 @@
 
```

Comment:
Nitpick that I should have caught before: can you rename this parser to something that has `json` in the name? If you can't name it just `json.py` because of conflicts with the `json` module that's fine but it would be nice to have something in the filename indicating that it's a generic json parser

Reply:
Sure, I can do the rename. 

The reason I named the module `generic`, is that I was also thinking of adding a generic `csv` parser in it as well. Since both classes are pretty simple I was thinking they could live in the same file/module.

That way you would import them as:
```python
from benchpress.plugins.parsers.generic import CSVParser
from benchpress.plugins.parsers.generic import JSONParser
```

Reply:
Ok that makes sense to me